### PR TITLE
[DNM] Add  fix your connection function

### PR DESCRIPTION
**Pull request by Pix1234**

### CODE REVIEW

File: sopel_channelmgnt/channelmgnt/__init__.py [hunk 1]
```diff
@@ -413,6 +413,54 @@ def invite_user(bot, trigger):
...
 
+@require_chanmsg
+@commands('fyc', 'fixconnection')
+@example('.fyc [#chan] user1 user!*@*')
+@priority('high')
+def fyckb(bot, trigger):
+    """Kick and ban a user from the channel, forwards user to specified channel until unbanned. The bot must be a channel operator for this command to work."""
+    chanops = get_chanops(str(trigger.sender), bot.memory["channelmgnt"]["jdcache"])
+    dodeop = False
+    forwardchan = "##fix_your_connection"
```
Comment (RhinosF1) [inline sopel_channelmgnt/channelmgnt/__init__.py]: Use a config variable

File: sopel_channelmgnt/channelmgnt/__init__.py [hunk 2]
```diff
@@ -413,6 +413,54 @@ def invite_user(bot, trigger):
...
+            time.sleep(1)
+            dodeop = True
+        text = trigger.group().split()
+        argc = len(text)
+        if argc < 3:
+            bot.reply('Syntax is: .fyc <nick> <reason>')
+            return
+        opt = Identifier(text[1])
+        nick = opt
+        mask = text[2] if any([s in text[2] for s in "!@*"]) else ''
```
Comment (RhinosF1) [inline sopel_channelmgnt/channelmgnt/__init__.py]: See deepsource

File: sopel_channelmgnt/channelmgnt/__init__.py [hunk 3]
```diff
@@ -413,6 +413,54 @@ def invite_user(bot, trigger):
...
+        channel = trigger.sender
+        reasonidx = 3 if mask != '' else 2
+        if not opt.is_nick():
+            if argc < 5:
+                bot.reply('Syntax is: .fyc <nick> <reason>')
+                return
+            channel = opt
+            nick = text[2]
+            mask = text[3] if any([s in text[3] for s in "!@*"]) else ''
```
Comment (RhinosF1) [inline sopel_channelmgnt/channelmgnt/__init__.py]: See deepsource

**Review by RhinosF1 — CHANGES_REQUESTED** [review]
See deepsource and sonar cloud

File: sopel_channelmgnt/channelmgnt/__init__.py (hunk 1 above)
Comment (Pix1234) [reply sopel_channelmgnt/channelmgnt/__init__.py]: Not sure how

Comment (RhinosF1) [reply sopel_channelmgnt/channelmgnt/__init__.py]: Look at the other defined options

File: sopel_channelmgnt/channelmgnt/__init__.py [hunk 4]
```diff
@@ -451,7 +452,7 @@ def fyckb(bot, trigger):
         if mask == '':
             mask = nick + '!*@*'
         if trigger.account in chanops:
-            bot.write(['MODE', channel, '+b', mask + '$' + forwardchan])
+            bot.write(['MODE', channel, '+b', mask + '${}'.format(bot.settings.channelmgnt.support_channel)])
```
Comment (RhinosF1) [inline sopel_channelmgnt/channelmgnt/__init__.py:455]: Don't use format. Not sure if it's enabled on this repo but the whole of core uses f strings.

Comment (Pix1234) [reply sopel_channelmgnt/channelmgnt/__init__.py:455]: How?

**Pull request closed without merging**